corehq/apps/app_manager/static/app_manager/js/app_exchange.js

```diff
@@ -1,4 +1,4 @@
-"use strict";
+
```

Comment:
nit: does the linter have a reason to want a blank line at the start of the file? Or is that just a side effect of removing the line?

Reply:
The latter. I don't know why it doesn't delete the whole, line, it's capable of doing other multi-line changes.